lib/data_import/kitsu.rb

```diff
@@ -41,22 +34,13 @@ def download_posters(ids)
       get_anime(ids) do |anime|
         file = Tempfile.new("anime_#{anime['id']}")
         file.binmode
-        req = get(anime['poster_image'].sub('https:', 'http:'))
+        req = get(anime['attributes']['posterImage']['original'].sub('https:', 'http:'))
```

Comment:
I think the probability is low, since its always rated by the most popular, and those always have the best data. But you are right, it's always better to be on the safer site.  :)